src/main/webapp/app/course/learning-paths/pages/learning-path-student-page/learning-path-student-page.component.html

```diff
@@ -1,12 +1,32 @@
 <div class="col learning-path-student-page">
-    @if (learningPathId()) {
+    @if (isLoading()) {
+        <div class="row justify-content-center align-items-center h-100">
+            <div class="spinner-border text-primary" role="status">
+                <span class="sr-only">Loading...</span>
```

Comment:
Should we add a translation here? We have `Laden...` in other places in Artemis

Reply:
The 'Loading...' is just for accessibility and is never shown. More infos on that here https://getbootstrap.com/docs/5.0/components/spinners/

Reply:
`sr-only` is for screen readers. A German user would hear an English text here, so I think we should localize it. I found some occurrences that look like this:

```jsx
                <span class="sr-only" jhiTranslate="loading"></span>
```